fastlane/lanes/release.rb

```diff
@@ -24,28 +24,25 @@
 
     release_branch_name = compute_release_branch_name(options:, version: release_version_next)
 
-    skip_user_confirmation = options[:skip_confirm]
+    # The `release_version_next` is used as the `new internal release version` value because the external and internal
+    # release versions are always the same.
+    message = <<~MESSAGE
+      Code Freeze:
+      • New release branch from #{DEFAULT_BRANCH}: #{release_branch_name}
 
-    unless skip_user_confirmation
-      # The `release_version_next` is used as the `new internal release version` value because the external and internal
-      # release versions are always the same.
-      message = <<-MESSAGE
```

Comment:
I want to call out in this diff the move from `<<-` to `<<~` and how it affects the indentation once `message` is printed.

![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/1218433/51c17faf-4e32-4e39-911c-9ee22f8910b1)

cc @spencertransier as (I think) the original author. It's a neat Ruby feature I hope you'll enjoy.